constructor_io/constructor_io.py

```diff
@@ -20,7 +21,9 @@ def __init__(self, api_token, autocomplete_key, protocol="https",
         self._protocol = protocol
         self._host = host
 
-    def _serialize_params(self, params):
+    def _serialize_params(self, params, sort=False):
```

Comment:
What is this sorting useful for?

Reply:
This is kinda a dirty hack :( We have such a test: https://github.com/Constructor-io/constructorio-python/blob/master/tests/test_constructor.py#L20
dict does not guarantee content ordering so sometimes content positioning switches and test fails.

Reply:
can you leave a comment to that effect? otherwise, next person working on this isn't going to know why the `sort=False` is there.